pom.xml

```diff
@@ -12,26 +12,27 @@
     <java.version>11</java.version>
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
-    <spring-boot.version>2.4.0</spring-boot.version>
-    <junit.version>5.7.0</junit.version>
-    <junit.platform.version>1.7.0</junit.platform.version>
-    <cucumber.version>6.9.1</cucumber.version>
-    <mockito.version>3.6.0</mockito.version>
-    <slf4j.version>1.7.30</slf4j.version>
-    <springdoc.version>1.5.2</springdoc.version>
-    <commons-lang3.version>3.11</commons-lang3.version>
-    <hsqldb.version>2.5.1</hsqldb.version>
-    <jakarta.xml.bind.version>2.3.3</jakarta.xml.bind.version>
-    <jboss-logging.version>3.4.1.Final</jboss-logging.version>
-    <lombok.version>1.18.16</lombok.version>
+    <spring-boot.version>2.5.2</spring-boot.version>
+    <junit.version>5.8.0-M1</junit.version>
+    <junit.platform.version>1.8.0-M1</junit.platform.version>
```

Comment:
try to use the latest version than milestone version. https://junit.org/junit5/

1.7.2